core/src/main/java/com/scalar/db/storage/jdbc/JdbcConfig.java

```diff
@@ -53,8 +53,11 @@ public class JdbcConfig {
       PREFIX + "mysql.variable_key_column_size";
   public static final String ORACLE_VARIABLE_KEY_COLUMN_SIZE =
       PREFIX + "oracle.variable_key_column_size";
+  public static final String DB2_VARIABLE_KEY_COLUMN_SIZE = PREFIX + "db2.variable_key_column_size";
   public static final String ORACLE_TIME_COLUMN_DEFAULT_DATE_COMPONENT =
       PREFIX + "oracle.time_column.default_date_component";
+  public static final String DB2_TIME_COLUMN_DEFAULT_DATE_COMPONENT =
+      PREFIX + "db2.time_column.default_date_component";
```

Comment:
The ScalarDB TIME type supports the microsecond fraction of a second, but the Db2 TIME type does not support fractional seconds. For this reason, I chose to use the Db2 TIMESTAMP type by default to map the ScalarDB TIME type. The drawback is that a fixed date component is stored with the TIME value, which increases the data volume.

Similarly to Oracle, the fixed date component value used to store value in the TIMESTAMP type is configurable with `scalar.db.jdbc.db2.time_column.default_date_component`